### DISCUSSION TITLE
What is Moddable Store?

### DISCUSSION THREAD

**Discussion by stc1988** [Moddable-OpenSource/moddable#677] [General]
Hi, I have a question in relation to documentation for [`mcbundle`.](https://github.com/Moddable-OpenSource/moddable/blob/61affe81dab4a8e6df64bd5ab32a0b39d758b3d4/documentation/tools/tools.md#mcbundle)

What is Moddable Store? Is there a application platform to distribute Moddable apps?

**Comment by phoddie** [comment]
Good question. It is an idea we had. We built `mcbundle` to help us explore that idea a bit.
